src/relay/pass/type_infer.cc

```diff
@@ -731,14 +800,11 @@ class TypeInferencer::Resolver : public ExprMutator, PatternMutator {
 Expr TypeInferencer::Infer(Expr expr) {
   // Step 0: Populate the constraints.
   GetType(expr);
-  // Step 1: Solve the constraints.
-  solver_.Solve();
 
-  if (err_reporter.AnyErrors()) {
-    err_reporter.RenderErrors(mod_);
-  }
+  // Step 2: Solve the constraints.
```

Comment:
Isn't the count off now? 😛

Reply:
yeah I've been adding and removing steps, experiments in progress.